lib/utils/fsUtils.js

```diff
@@ -156,73 +156,23 @@ function configExists(templateType, handle) {
 }
 
 function readConfig(templateType, handle) {
-  createConfigIfMissing(templateType, handle);
-  const configPath = path.join(
-    process.cwd(),
-    `${FOLDERS[templateType]}`,
-    `${handle}`,
-    "config.json"
-  );
-  const configData = fs.readFileSync(configPath).toString();
-  return JSON.parse(configData);
-}
+  try {
+    const configPath = path.join(
+      process.cwd(),
+      `${FOLDERS[templateType]}`,
+      `${handle}`,
+      "config.json"
+    );
 
-/**
```

Comment:
Not sure why we are removing the entire creation of config, this will not only affect the reading of configs